packages/now-cli/src/util/dev/server.ts

```diff
@@ -1706,7 +1710,29 @@ export default class DevServer {
 
     this.output.debug(`Spawning dev command: ${command}`);
 
-    const p = spawnCommand(command, { stdio: 'inherit', cwd, env });
+    const p = spawnCommand(command, {
+      stdio: [process.stdin, 'pipe', 'pipe'],
+      cwd,
+      env,
+    });
+
+    if (!p.stdout || !p.stderr) {
+      throw new Error('Expected child process to have stdout and stderr');
+    }
+
+    p.stdout.on('data', data => {
+      if (data) {
+        this.output.debug(`Replacing framework output with now dev address`);
+        data = data
+          .toString()
+          .replace(`http://localhost:${port}`, this.address);
+      }
+      process.stdout.write(data);
+    });
+
+    p.stderr.on('data', data => {
```

Comment:
Why not use `.pipe()`?

```suggestion
    p.stderr.pipe(process.stderr)
```

Reply:
In case we wanted to transform stderr too?

Reply:
Fixed